### PR TITLE
Add sorting to grids columns

### PR DESCRIPTION
**Pull request by joseguti84**

### CODE REVIEW

File: source/CommonJobs/CommonJobs.MVC.UI/Scripts/evaluations-admin.js [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
         if (data) {
             this.fromJS(data);
         }
+        this.headers = [
+        { title: 'Empleado', sortPropertyName: 'fullName', asc: true},
+        { title: 'Puesto', sortPropertyName: 'currentPosition', asc: true},
+        { title: 'Seniority', sortPropertyName: 'seniority', asc: true},
+        { title: 'Responsable', sortPropertyName: 'responsible', asc: true}
+        ];
+        this.activeSort;
```
Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/evaluations-admin.js]: What is this line?

Comment (andresmoschini) [inline source/CommonJobs/CommonJobs.MVC.UI/Scripts/evaluations-admin.js]: Oh, I understand now, could you assign them a default parameter to make easier to read it?

**Comment by andresmoschini** [comment]
We have discussed with @joseguti84 in person.

He will move `activeSort` to a flag in each header in order to have better encapsulation and allow the view to know about header state.

Also, he will extract common sorting functionality in a function.

**Comment by joseguti84** [comment]
Thanks @andresmoschini !

**Comment by andresmoschini** [comment]
Nice!

**Pull request merged**